include/storage/FileSystem.h

```diff
@@ -32,10 +32,15 @@ class FileSystem
 
     static void clearSharedFiles();
 
+    std::string getPathForFd(int fd);
+
   private:
     int nextFd;
+
     std::unordered_map<int, storage::FileDescriptor> fileDescriptors;
 
+    std::unordered_map<int, std::string> fdNames;
```

Comment:
This is to make the logging around filesystem calls easier. Before it was just logging integers for file descriptors which was a little difficult to follow. Now it logs the path of the file by looking it up in this map.